packages/opencensus-core/src/trace/model/types.ts

```diff
@@ -30,6 +30,17 @@ export interface Attributes {
   [attributeKey: string]: string|number|boolean;
 }
 
+/**
+ * The status of a Span by providing a standard code in conjunction
+ * with an optional descriptive message.
+ */
+export interface Status {
+  /** The status code. */
+  code: number;
+  /** A developer-facing error message. */
+  message?: string;
+}
+
```

Comment:
What would you think about adding an enum with the list of OpenCensus status codes from the table referenced above?

Reply:
To be clear I'm not suggesting the type of the `code` field be the enum (that might limit clients of the library if a new status is added but they can't upgrade yet, and for `opencensus-web` I want to keep `@opencensus/core` just as a dev dependency). But just having the enum provides documentation for what the possible canonical status codes are.

Reply:
I checked in Java implementation, we are using enum for ```code``` -> https://github.com/census-instrumentation/opencensus-java/blob/master/api/src/main/java/io/opencensus/trace/Status.java#L384. This is also matching with [status spec](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/HTTP.md#status).

Reply:
OK, I'd be fine with using an enum for it.